test/navi/core_test.clj

```diff
@@ -185,13 +213,22 @@
           hparam    (doto (HeaderParameter.)
                       (.setName "y")
                       (.setSchema (StringSchema.)))
+          response  (doto (ApiResponse.)
+                      (.setContent (doto (Content.)
+                                     (.put "application/json"
+                                           (doto (MediaType.)
+                                             (.setSchema (ObjectSchema.)))))))
+          responses (doto (ApiResponses.)
+                      (.put "200" response))
```

Comment:
is an int possible here?

Reply:
If you mean instead of the string "200", then no, as I explained above.  I tried to `.put` and int and got an exception.

Reply:
All good.